### ISSUE TITLE
Helper Commands

### ISSUE THREAD

**Issue by 2016rshah** [afg984/pyardrone#1]
It would be nice if you could have helper methods like `drone.forward()` rather than having to say stuff like `drone.move(pitch=-0.5)`. After all, the main functionality of the drone is moving it around, so I think the library should make that as easy as possible. If you think its a good idea, I would be happy to help you out with adding that functionality

**Comment by afg984** [comment]
Since takeoff and land helper functions are implemented, it would be nice to have functions like forward available.

see also:
node-ar-drone https://github.com/felixge/node-ar-drone
stores speed in the client object and use it as the speed for helper functions like forward, up, down ....

**Comment by afg984** [comment]
Do you think storing speed in the drone object itself is a good idea?
Also, how many helpers should we implement?
Should ``backward()`` be a function itself or ``forward()`` with negative speed?

**Comment by 2016rshah** [comment]
I would say `forward()`, `backward()`, `clockwise()`, and `counterclockwise()` would be a good place to start. That will cover most of the bases, and if we want to implement more later, we can. 

As far as speed goes, I don't really know. What do you think?

I definitely think it is better to define both directions separately because that will simplify reasoning about direction of the drone. You don't think about the drone going negative forward, you think about it going backwards, right?

I can get started working on the features early next week, if that's alright with you

**Comment by afg984** [comment]
``forward()``, ``backward()``, ``clockwise()``, and ``counterclockwise()`` functions sound reasonable.

I did not implement them because I have to shift and rotate at the same time in my use case.

Feel free to file a pull request if you'd like to.

About the speed...

* Required
    ```python
    def forward(self, speed):
        self.move(pitch=speed)
    ```
    ``drone.forward(0.7)`` seems to have the same verbosity as ``drone.move(pitch=0.7)``, with the exception of being more readable.

* Optional, with hard-coded default value
    ```python
    def forward(self, speed=1):
        self.move(pitch=speed)
    ```
    Providing a reasonable default value is quite challenging.

* Optional, with default value stored in ``ARDrone`` object
    ```python
    def forward(self, speed=None):
        self.move(pitch=speed if speed is not None else self.speed)
    ```
    I'm wondering whether it is too much for the ``ARDrone`` to maintain another vairable which has nothing to do with the control flow.

Any ideas?

**Comment by 2016rshah** [comment]
I like the idea of a hard coded default value (option two) because it gives the user freedom to not provide a speed but the flexibility to fine tune the speed if they want. I'll get started as soon as possible.

**Comment by afg984** [comment]
I like that one the best too.

**Comment by afg984** [comment]
@2016rshah Please give the pull request a review if you have time, thanks!

**Comment by afg984** [comment]
#7 merged, v0.5.0 uploaded to pypi.
